docs/modules/ROOT/pages/get-started.adoc

```diff
@@ -201,6 +201,8 @@ You can monitor the Hazelcast cluster by starting Management Center. You can cre
 include::ROOT:example$/management-center.yaml[]
 ----
 
+WARNING: `licenseKeySecret` field is not required, if your Hazelcast cluster is open source. Please remove or comment out this line.
```

Comment:
I wouldn't say this is a warning. A warning is something serious such as some config that can lead to a failure.

Do you think it would be better to have a `management-center-enterprise.yaml` file that includes this field? That way, we can use the Open Source/Enterprise tabs and avoid this note altogether.

Reply:
good idea 👍

@JakeSCahill I applied the recommendation but it failed as you can see below and I can not understand the reason 🙂